source/selinux/selinux.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package selinux
+
+import (
+	"io/ioutil"
+	"github.com/golang/glog"
+)
+
+type Source struct{}
+
+func (s Source) Name() string { return "selinux" }
+
+func (s Source) Discover() ([]string, error) {
+	features := []string{}
+	status, err := ioutil.ReadFile("/sys/fs/selinux/enforce")
```

Comment:
If we mount this in a different path as per my comment above, this should be modified.

Reply:
Is checking the contents of `enforce` sufficient to determine the status of selinux? Do we need to examine the contents of `status` as well?